cryptoinvestor/views/assets.py

```diff
@@ -42,6 +45,112 @@ def get_objects(self):
                 }
             }
 
+        def dispatch_request(self):
+            if request.method == 'POST':
+                print("test")
+
+        graphs_json = json.dumps(graphs, cls=plotly.utils.PlotlyJSONEncoder)
+
+        return {
+            'assets': assets,
+            'local_currency': self.app.local_currency,
+            'graphJSON': graphs_json
+        }
+
+
+class Sell(BaseView):
```

Comment:
So you just basically copied `AssetsListView` 2 times and have the same views on 3 different places with different URLs. This is a terrible idea. There should be only one view with assets and buy and sell views should just handle all the necessary data from request call appropriate model methods and redirect to the asset views.

Reply:
done